sdk/core/azure-common/azure/common/client_factory.py

```diff
@@ -22,10 +22,10 @@
 
 
 def _instantiate_client(client_class, **kwargs):
-    """Instantiate a client from kwargs, removing the subscription_id/tenant_id/base_url argument if unsupported.
+    """Instantiate a client from kwargs, removing unsupported stuff.
```

Comment:
I wouldn't say "stuff" in comments.  Please be more explicit

Reply:
That's hidden method, it's why I wasn't very verbose, but yeah can make it more explciit :)

Reply:
Fixed